libff/algebra/curves/bls12_381/bls12_381_init.cpp

```diff
@@ -0,0 +1,634 @@
+#include <libff/algebra/curves/bls12_381/bls12_381_g1.hpp>
+#include <libff/algebra/curves/bls12_381/bls12_381_g2.hpp>
+#include <libff/algebra/curves/bls12_381/bls12_381_init.hpp>
+
+namespace libff
+{
+
+bigint<bls12_381_r_limbs> bls12_381_modulus_r; // (VV)
+bigint<bls12_381_q_limbs> bls12_381_modulus_q; // (VV)
+
+bls12_381_Fq bls12_381_coeff_b;
+bigint<bls12_381_r_limbs> bls12_381_trace_of_frobenius; // from bls12_377 (VV)
+bls12_381_Fq2 bls12_381_twist;
+bls12_381_Fq2 bls12_381_twist_coeff_b;
+bls12_381_Fq bls12_381_twist_mul_by_b_c0;
```

Comment:
I think all of these endmorphism and "untwist-frobnius-twist" related values can be removed now.